salt/elife-dashboard/scheduler.sls

```diff
@@ -25,6 +25,11 @@ install-{{ app.name }}:
         - require:
             - git: install-{{ app.name }}
 
+    cmd.run:
+        - cwd: /srv/{{ app.name }}
+        - name: ./install.sh
```

Comment:
weird that this wasn't done before?

Reply:
yeah ... we stopped including `branch` in the buildvars a while back and started using `revision` or similar recently. I'm seeing lots of little warts I want to fix as I go along but it's just what is necessary to get a vagrant machine booted to working order on 1604 for now.